src/OpenColorIO/transforms/FileTransform.cpp

```diff
@@ -985,10 +985,33 @@ OIIO_ADD_TEST(FileTransform, AllFormats)
     OIIO_CHECK_ASSERT(FormatExtensionFoundByName("spi3d", "spi3d"));
     OIIO_CHECK_ASSERT(FormatExtensionFoundByName("spimtx", "spimtx"));
     OIIO_CHECK_ASSERT(FormatExtensionFoundByName("vf", "nukevf"));
+}
+
+void GetFormatByIndex(OCIO::FormatRegistry &reg, int cap)
```

Comment:
[Minor] The method is doing validations so a better name could be 'ValidateFormats'

Reply:
Updated the name.